src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -257,6 +258,28 @@ def load_arguments(self, _):
         c.argument('nodepool_name', type=str,
                    help='Node pool name, upto 12 alphanumeric characters', validator=validate_nodepool_name)
 
+    with self.argument_context('aks nodepool') as c:
+        c.argument('cluster_name', type=str, help='The cluster name.')
+
+    for scope in ['aks nodepool add']:
+        with self.argument_context(scope) as c:
+            c.argument('nodepool_name', type=str, options_list=['--name', '-n'], validator=validate_nodepool_name, help='The node pool name.')
+            c.argument('zones', zones_type, options_list='--zones', help='Space-separated list of availability zones where agent nodes will be placed.')
```

Comment:
Maybe you can add an alias `-z` just like `vm/vmss create` to provide consistent experience across commands.

Reply:
Done